kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/usagelimits/ratelimits/UsageHistory.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.kotlindiscord.kord.extensions.usagelimits.ratelimits
+
+/** Interface that provides variables that hold usage history information about command usage and rate-limits. **/
```

Comment:
Now we have a dash in rate-limits? This is correct, but you didn't do that in any other references to the term above.